Lib/test/test_opcodes.py

```diff
@@ -41,15 +41,6 @@ def test_use_existing_annotations(self):
         exec('x: int', ns)
         self.assertEqual(ns['__annotations__'], {'x': 'int', 1: 2})
 
-    def test_do_not_recreate_annotations(self):
```

Comment:
This test is not valid anymore because all annotations values will be set within `SETUP_ANNOTAITONS` opcode.